app/controllers/sessions_controller.rb

```diff
@@ -4,13 +4,21 @@ def new
 
   def create
     user = User.find_by(email: params[:email])
-    # require "pry"; binding.pry
     if user != nil && user.authenticate(params[:password])
       session[:user_id] = user.id
       flash[:success] = "Welcome, #{user.name}!"
     end
-    if user.merchant?
+
+    if user.default?
+      redirect_to '/profile'
+      flash[:success] = "#{user.name} is logged in!"
+    elsif user.merchant?
       redirect_to '/merchant/dashboard'
+      flash[:success] = "#{user.name} is logged in!"
+    elsif user.admin?
+      redirect_to '/admin/dashboard'
+      flash[:success] = "#{user.name} is logged in!"
     end
```

Comment:
This conditional redirects to the appropriate show pages with the flash message displayed. We need to talk about namespacing some of our resources and how we want to use the dashboard page.